server/src/test/java/org/elasticsearch/rest/action/document/RestGetActionTests.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.rest.action.document;
+
+import org.elasticsearch.action.get.GetRequest;
+import org.elasticsearch.action.get.GetResponse;
+import org.elasticsearch.common.RestApiVersion;
+import org.elasticsearch.rest.RestRequest;
+import org.elasticsearch.test.rest.FakeRestRequest;
+import org.elasticsearch.test.rest.RestActionTestCase;
+import org.junit.Before;
+import org.mockito.Mockito;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.instanceOf;
+
+public class RestGetActionTests extends RestActionTestCase {
```

Comment:
hmm... this is fail when master is on version 9.  Can we add an assumeTrue(RestApiVersion.current() == 8) ..or something like that, also with a note to delete in v9+

Reply:
Will `RestApiVersion.V_7` exist in master at all?  Often dead code like this is removed on upgrade because the constants that it uses for guards are deleted, in which case this test class will fail to compile.

Reply:
@romseygeek that is the plan. Once the `RestApiVersion.V_7` is deleted, we will delete the tests using it.
However we don't want to create a blocker during the upgrade process and we expect for a brief period of time RestApiVersion will have V_7 (obsolete) V_8 compatible and V_9 current.